src/wallet/wallet.cpp

```diff
@@ -2566,9 +2566,10 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
                 // to avoid conflicting with other possible uses of nSequence,
                 // and in the spirit of "smallest posible change from prior
                 // behavior."
+                bool rbf = (flags & CREATE_TX_ENABLE_RBF || ::fWalletRbf) && !(flags & CREATE_TX_DISABLE_RBF);
```

Comment:
This would be more readable as:

```C++
bool rbf = ::fWalletRbf;
if (flags & CREATE_TX_ENABLE_RBF) {
    rbf = true;
} else if (flags & CREATE_TX_DISABLE_RBF) {
    rbf = false;
}
```